configserver/src/main/java/com/yahoo/vespa/config/server/ConfigServerBootstrap.java

```diff
@@ -100,6 +129,8 @@ private ConfigServerBootstrap(ApplicationRepository applicationRepository, RpcSe
         this.sleepTimeWhenRedeployingFails = Duration.ofSeconds(configserverConfig.sleepTimeWhenRedeployingFails());
         this.exitIfRedeployingApplicationsFails = exitIfRedeployingApplicationsFails;
         rpcServerExecutor = Executors.newSingleThreadExecutor(new DaemonThreadFactory("config server RPC server"));
+        prepareAllAppsBeforeActivating = Flags.PREPARE_ALL_APPS_BEFORE_ACTIVATING_AT_BOOTSTRAP
+                .bindTo(flagSource).with(FetchVector.Dimension.ZONE_ID, applicationRepository.zone().toString());
```

Comment:
There is seldom any need to specify the zone dimension as all flags can be overridden per zone.  (The only reason I can think of is in the controller, and if it wants to do things differently per zone.)

Reply:
Good point, will remove zone dimension.